app/locale/ja.coffee

```diff
@@ -336,7 +336,7 @@ module.exports = nativeDescription: "日本語", englishDescription: "Japanese",
     tip_google: "解決できない問題がありますか？ググっちゃえ！"
     tip_adding_evil: "悪を少々加える。"
     tip_hate_computers: "コンピュータを憎むと思う人についての事だ。彼らが本当に嫌いなのは下手なプログラマーだ。- ラリー・ニーヴン"
-#    tip_open_source_contribute: "You can help CodeCombat improve!"
+    tip_open_source_contribute: "あなたはCodeCombatを向上させることができます！"
 #    tip_recurse: "To iterate is human, to recurse divine. - L. Peter Deutsch"
```

Comment:
It is something like: "Iterating is human, recursing is divine.". It has programming concepts involved.